sql/2021/caching/cookie_attributes.sql

```diff
@@ -0,0 +1,36 @@
+CREATE TEMPORARY FUNCTION getCookieAttributes(headers STRING) RETURNS ARRAY<STRING> DETERMINISTIC LANGUAGE js AS '''
```

Comment:
Nit but quite a long line. Should we prefer this:

```suggestion
CREATE TEMPORARY FUNCTION getCookieAttributes(headers STRING)
RETURNS ARRAY<STRING> DETERMINISTIC LANGUAGE js
AS '''
```

Also no `#standardSQL` or comment.